static/app/components/timeSince.tsx

```diff
@@ -169,14 +169,12 @@ export function getRelativeDate(
   if ((shorten || extraShort) && !suffix) {
     return getDuration(moment().diff(moment(date), 'seconds'), 0, shorten, extraShort);
   }
-  if (!suffix) {
-    return moment(date).fromNow(true);
-  }
   if (suffix === 'ago') {
     return moment(date).fromNow();
   }
-  if (suffix === 'old') {
-    return t('%(time)s old', {time: moment(date).fromNow(true)});
+  if (suffix?.length) {
```

Comment:
nit, but you could keep the early return from L172 and you wouldnt need to check for length, just return with a suffix instead of throwing in the end. Not a big deal anyways